Modules/SubModule_Import/xASL_imp_Initialize.m

```diff
@@ -75,7 +77,12 @@
     if ~isfield(imPar, 'sessionNames') || isempty(imPar.sessionNames)
         imPar.sessionNames = {};
     end
-
+    
+    % Create the empty derivatives directory for the general import (we need it for lock and temp files)
```

Comment:
Note that `xASL_adm_CreateDir` not only creates the folder you want to create but also folder layers above it. So you don't have to first create `/derivatives` and then `/derivatives/lock`.

Furthermore, is `lock` not already created by the general mutex script(s)?

Reply:
@HenkMutsaerts Yes. But note that this was limited to two layers only, right?
We have root/derivatives/ExploreASL/lock - so it makes sense to have two commands because there are three layers.

Reply:
No, that's one main feature of `xASL_adm_CreateDir`, beside for checking if the folder already exists:

>> newFolder = fullfile(pwd,'layer1','layer2','layer3','layer4');
>> xASL_adm_CreateDir(newFolder);

This creates all layers.